docs/RuntimeAttributes.md

```diff
@@ -186,46 +220,67 @@ runtime {
 }
 ```
 
-## `bootDiskSizeGb`
+### `docker`
+
+When specified, Cromwell will run your task within the specified Docker image. 
 
-In addition to working disks, Google Cloud allows specification of a boot disk size. This is the disk where the docker image itself is booted (**not the working directory of your task on the VM**).
-Its primary purpose is to ensure that larger docker images can fit on the boot disk.
 ```
 runtime {
-  # Yikes, we have a big OS in this docker image! Allow 50GB to hold it:
-  bootDiskSizeGb: 50
+  docker: "ubuntu:latest"
 }
 ```
 
-Since no `local-disk` entry is specified, Cromwell will automatically add `local-disk 10 SSD` to this list.
+- Local: Cromwell will automatically run the docker container.
+- SFS: When a docker container exists within a task, the `submit-docker` method is called. See the [Getting started with containers](/tutorials/Containers/) guide for more information.
+- GCP: This attribute is mandatory when submitting tasks to Google Cloud.
+- AWS Batch: This attribute is mandatory when submitting tasks to AWS Batch.
 
-## `zones`
 
-The ordered list of zone preference (see [Region and Zones](https://cloud.google.com/compute/docs/zones) documentation for specifics).
+### `maxRetries`
 
-*The zones are specified as a space separated list, with no commas:*
+*Default: _0_*
+
+This retry option is introduced to provide a strategy for tackling transient job failures. For example, if a task fails due to a timeout from accessing an external service, then this option helps re-run the failed the task without having to re-run the entire workflow. It takes an Int as a value that indicates the maximum number of times Cromwell should retry a failed task. This retry is applied towards jobs that fail while executing the task command. 
```

Comment:
I think it's worth mentioning that these are dumb retries (the word "strategy" is what triggered the thought) instead of e.g. increasing memory due to OOM situations

Reply:
This was one that we have moved but haven't changed, but happy to add some additional clarification. I didn't want to say dumb, so I've went for this sentence at the end (8cc7be9984f9ffad102ba2ad62e91925ef0caeb0):

> This method only applies to transient job failures and is a feeble attempt to retry a job, that is it cannot be used to increase memory in out-of-memory situations.